src/esn.calendar.libs/app/components/select-calendar/select-calendar-item.less

```diff
@@ -8,6 +8,14 @@
 
   .info {
     .flex;
+    .name{
+      .ellipsis(); 
+
+      max-width: 250px;
```

Comment:
I don't like to have a width fixed, can we have something better with the parent size or better and more simple max-width: 100vw

Reply:
i will use limit to filter in pug file to avoid the problem of width .Simple and elegant solution!